.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
```

Comment:
This new workflow only triggers when there's a push to `master` (we want to deploy to TestPyPI) or a new release has been published (we want to deploy to production PyPI)

Reply:
✌️✌️